### PR TITLE
Feat: Style Hero PLP

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?
This PR aims to adapt the `Hero` component so that we can use it on the `plp` page as well.
Now we can pass new props: 
- `icon`
- `variants`: `small` and `default`.

## Preview
|Homepage (variant = default)|PLP (variant = small)|
|-|-|
|homehero|plp|
## How to test it
👀   Preview: https://preview-230--base.preview.vtex.app

## References
[FSSS-114](https://vtex-dev.atlassian.net/jira/software/projects/FSSS/boards/632?selectedIssue=FSSS-114)

### CODE REVIEW

**Review by filipewl — APPROVED** [review]

File: src/components/ui/Hero/hero.scss [hunk 1]
```diff
@@ -63,23 +48,69 @@
...
+
+      @include media(">=tablet") {
+        width: 50%;
+        padding: var(--space-5);
+      }
+
+      @include media(">=notebook") { padding-left: var(--space-7); }
+    }
+
+    .title-hero {
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:104]: Niceeee!

File: src/components/ui/Hero/hero.scss [hunk 2]
```diff
@@ -63,23 +48,69 @@
...
+      font-size: var(--space-5);
+
+      @include media(">=tablet") { font-size: var(--space-7); }
+    }
+
+    p {
+      font-size: var(--space-3);
+
+      @include media(">=tablet") { font-size: 1.25rem; }
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:113]:
```suggestion
      @include media(">=tablet") { font-size: var(--text-size-3); }
```

File: src/components/ui/Hero/hero.scss [hunk 3]
```diff
@@ -63,23 +48,69 @@
...
+    }
+
+    .title-hero {
+      font-size: var(--space-5);
+
+      @include media(">=tablet") { font-size: var(--space-7); }
+    }
+
+    p {
+      font-size: var(--space-3);
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:111]:
```suggestion
      font-size: var(--text-size-2);
```

File: src/components/ui/Hero/hero.scss [hunk 4]
```diff
@@ -63,23 +48,69 @@
...
+        padding: var(--space-5);
+      }
+
+      @include media(">=notebook") { padding-left: var(--space-7); }
+    }
+
+    .title-hero {
+      font-size: var(--space-5);
+
+      @include media(">=tablet") { font-size: var(--space-7); }
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:107]:
```suggestion
      @include media(">=tablet") { font-size: var(--text-size-7); }
```

File: src/components/ui/Hero/hero.scss [hunk 5]
```diff
@@ -63,23 +48,69 @@
...
+      @include media(">=tablet") {
+        width: 50%;
+        padding: var(--space-5);
+      }
+
+      @include media(">=notebook") { padding-left: var(--space-7); }
+    }
+
+    .title-hero {
+      font-size: var(--space-5);
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:105]:
```suggestion
      font-size: var(--text-size-5);
```

**Review by lucasfp13 — APPROVED** [review]
After review and apply the code suggestions, LGTM!

**Review by hellofanny — CHANGES_REQUESTED** [review]
Some adjusts and LGTM! 🚀 🚀 🚀

File: src/components/ui/Hero/hero.scss (hunk 2 above)
Comment (eduardoformiga) [reply src/components/ui/Hero/hero.scss]: ohhh my, completely forgot these typography variables, hahaha.

**Comment by eduardoformiga** [comment]
> 

Completely forgot these typography variables! Good catch! 😄

**Comment by hellofanny** [comment]
I also noticed something else: 🤔 

|Before|
|-|
image

|After|
|-|
image

**Comment by eduardoformiga** [comment]
> I also noticed something else: 🤔 Can you please check this
> 
> Browser: Chrome Size: 1920x1080
> [...] 


@hellofanny. Thank you for the heads up!
I changed the implementation now focusing on keeping the margin and padding following the hole page as a reference.
I also updated the images in the description considering both variants.

File: src/styles/typography.scss [hunk 6]
```diff
@@ -28,14 +28,6 @@ h1, h2, h3, h4, h5, h6, p, ul {
...
 // Titles ///////////////////////////
 
-.title-hero {
-  font-weight: var(--text-weight-black);
-  font-size: var(--space-5);
-  line-height: 1.1;
-
-  @include media(">=notebook") { font-size: var(--space-8); }
-}
-
```
Comment (renatamottam) [inline src/styles/typography.scss:38]: Don't know if I like removing it from here, it's useful for future customization to see all the major typography sizes on the same page.

File: src/components/ui/Hero/hero.scss [hunk 7]
```diff
@@ -6,7 +6,7 @@
   width: 100%;
   background-color: var(--bg-secondary-default);
 
-  @include media(">=notebook") {
+  @include media(">=tablet") {
```
Comment (renatamottam) [inline src/components/ui/Hero/hero.scss:9]: I have an idea to fix this, slack me!

File: src/styles/typography.scss (hunk 6 above)
Comment (hellofanny) [reply src/styles/typography.scss:38]: Hm! That is something to consider.
So, just let's keep it here? 👀

**Review by hellofanny — APPROVED** [review]
Well done!! 🚀 🚀 🚀 After consider what renata said I think we are ready to go!

File: src/styles/typography.scss (hunk 6 above)
Comment (eduardoformiga) [reply src/styles/typography.scss]: We decided together to keep this on typography and adapt in hero. 🎉

**Comment by filipewl** [comment]
@renatamottam is going to make adjustments later and [said](https://vtex.slack.com/archives/C02MVS48Y0Z/p1642164807328100?thread_ts=1642162598.324800&cid=C02MVS48Y0Z) we could merge this!

**Pull request merged**